umm-spec-lib/src/cmr/umm_spec/migration/related_url_migration_maps.clj

```diff
@@ -160,3 +160,93 @@
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "ANOMALIES"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "CASE STUDY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "DATA CITATION POLICY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "DATA QUALITY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "DATA RECIPE"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "DELIVERABLES CHECKLIST"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "GENERAL DOCUMENTATION"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "HOW-TO"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "IMPORTANT NOTICE"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "INSTRUMENT/SENSOR CALIBRATION DOCUMENTATION"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "MICRO ARTICLE"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PI DOCUMENTATION"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PROCESSING HISTORY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PRODUCT HISTORY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PRODUCT QUALITY ASSESSMENT"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PRODUCT USAGE"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PRODUCTION HISTORY"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "PUBLICATIONS"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "READ-ME"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "REQUIREMENTS AND DESIGN"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "SCIENCE DATA PRODUCT SOFTWARE DOCUMENTATION"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "SCIENCE DATA PRODUCT VALIDATION"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "USER FEEDBACK PAGE"}
+   {:URLContentType "PublicationURL" :Type "VIEW RELATED INFORMATION" :Subtype "USER'S GUIDE"}
+   {:URLContentType "VisualizationURL" :Type "GET RELATED VISUALIZATION"}
+   {:URLContentType "VisualizationURL" :Type "GET RELATED VISUALIZATION" :Subtype "GIOVANNI"}
+   {:URLContentType "VisualizationURL" :Type "GET RELATED VISUALIZATION" :Subtype "MAP"}
+   {:URLContentType "VisualizationURL" :Type "GET RELATED VISUALIZATION" :Subtype "WORLDVIEW"}])
+
+(def umm-1-12-umm-url-types->umm-1-11-umm-url-types
+  "Mapping from the UMM-C v1.11 URLContentType, Type, and Subtype to the UMM v 1.10 URLContentType, Type, and Subtype
```

Comment:
Update the docstring to say v1.12 and v1.11.

Reply:
done